manifest/armv7l/g/gimp.filelist

```diff
@@ -1,16 +1,11 @@
-/usr/local/bin/gimp
 /usr/local/bin/gimp-2.99
```

Comment:
Does this mean in order to start we have to type gimp-2.99?  Probably should restore if these are symlinks.

Reply:
If you hit gimp <tab> it autocompletes to gimp-2.99.

As such, should we add symlinks to gimp or not?